### ISSUE TITLE
Manual Sorting in Crates Like in Playlists with Column #

### ISSUE THREAD

**Issue by Russe** [mixxxdj/mixxx#11757]
### Feature Description

Within playlists there's a column "#", and when sorting for this column it's possible to move tracks up an down.
So this is like sorting real records in a crate.

But within crates there's no column "#", so I can not sort a crate like I want to during a gig.
I would like to do this also with crates, because a crate can contain a specific track only once.

So I would like to ask for implementing this feature into Mixxx?

**Comment by ronso0** [comment]
So you're using crates like playlists?
Use playlists in the first place :)

Unlikely this will be implemented as this doesn't match the Crate concept, and it's a lot if work.

**Comment by Russe** [comment]
Yes, currently I'm using playlists for this because with crates it's not possible.

Why would this does not match the crate concept?
When looking into real live, when doing a gig with vinyl records, these records are in my record case, each record only once. During this gig I always browse my records and move these records to the front which I want to play next.

In my opinion, a playlist has a different approach, individual tracks can be in playlist multiple times.

At the end, it would be nice to have either crates with column "#" for manual sorting, or playlists with an additional feature to allow each track to be in this playlist only once.
I would vote for enhancement of crates in this case, because for me it's similar to vinyl records at a record case.

**Comment by ronso0** [comment]
You're right & and I understand the use case.
> or playlists with an additional feature to allow each track to be in this playlist only once.

this would complicate UX, so adding # to crates is probabyl preferred.

**Comment by Russe** [comment]
As #5634 seems to be implemented in near future it would be awesome for me to have column # at crates. Because than I could get rid of most of my playlists and move them for better handling to crates section.
I can not implement # at crates by myself because I'm not a coder, but I'll assist with testing or something similar. So when someone is working on this I'll step into.

**Comment by ronso0** [comment]
> As https://github.com/mixxxdj/mixxx/issues/5634 seems to be implemented in near future

Works has just been started and Mixxx 2.6 will reach feature freeze soonish, so that's unlikely to happen 'soon'. Earliest I expect is Mixxx 2.7 (or whatever that version will be called, maybe Mixxx 3.0 if the QML is polished then).

**Comment by Russe** [comment]
That's a pity. But thanks for clarification.
